### PR TITLE
feat(filter): accordion redesign

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: [#568](https://github.com/unimelb/ffam-prototype/issues/568)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit b9808c0e1be83f627a7095b0381c8f862f27e317

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5d6757dda89db900088301e1

**Comment by tomstringer** [comment]
Hmm. Still looking the same in Matrix...

See: https://finearts-music.unimelb.edu.au/_web_services/ajax-version/funnelback/dev/raw-code-test/_recache?dev=latest

Screen Shot 2019-08-29 at 2 50 20 pm

**Comment by gepifanio** [comment]
> Hmm. Still looking the same in Matrix...
> 
> See: https://finearts-music.unimelb.edu.au/_web_services/ajax-version/funnelback/dev/raw-code-test/_recache?dev=latest
> 
> Screen Shot 2019-08-29 at 2 50 20 pm

Should be this link to be tested:

https://deploy-preview-1267--pattern-lib-unimelb.netlify.com/?selectedKind=Filter%20%2F%20Components&selectedStory=Accordion%20Filter&full=1&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

File: components/filter/accordion/AccordionFilter.vue [hunk 1]
```diff
@@ -80,42 +80,40 @@
...
-              <tr>
-                <td colspan="5">
-                  {{ item.overview }}
-                  <br>
-                  <ButtonIcon size="xsml">{{ item.buttonText }}</ButtonIcon>
-                </td>
-              </tr>
+            <table class="table table--striped table--tight table--is-compacted">
+              <tbody>
+                <col width="30%">
```
Comment (TimMansell) [inline components/filter/accordion/AccordionFilter.vue]: I think we should wrap `<col>`'s in `<colgroup>` even though it is optional.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/colgroup

**Pull request merged**